v3data/hypes/fees_yield_data.py

```diff
@@ -212,7 +213,7 @@ async def _get_pool_data_at_block(
 
         response = await self.uniswap_client.query(pool_query, variables)
 
-        return response["data"]
+        return response.get("data", {})
```

Comment:
Lets add some logging in cases such as here where it falls back to the default.  I think it's a reasonable change to make sure endpoint doesn't crash - but might make debugging hard when empty results start showing up without logging.

Reply:
added extensive logging